src/wrapper/objects/auto_array.rs

```diff
@@ -64,16 +64,16 @@ type_array!(jdouble, GetDoubleArrayElements, ReleaseDoubleArrayElements);
 ///
 /// AutoArray provides automatic array release through a call to appropriate
 /// Release<Type>ArrayElements when it goes out of scope.
-pub struct AutoArray<'a: 'b, 'b, T: TypeArray> {
+pub struct AutoArray<'a, T: TypeArray> {
     obj: JObject<'a>,
     ptr: NonNull<T>,
     mode: ReleaseMode,
     is_copy: bool,
-    env: &'b JNIEnv<'a>,
+    env: JNIEnv<'a>,
 }
 
-impl<'a, 'b, T: TypeArray> AutoArray<'a, 'b, T> {
-    pub(crate) fn new(env: &'b JNIEnv<'a>, obj: JObject<'a>, mode: ReleaseMode) -> Result<Self> {
+impl<'a, T: TypeArray> AutoArray<'a, T> {
+    pub(crate) fn new(env: JNIEnv<'a>, obj: JObject<'a>, mode: ReleaseMode) -> Result<Self> {
```

Comment:
We can just use a "plain" (i.e. without lifetime marker) reference here.
```suggestion
    pub(crate) fn new(env: &JNIEnv<'a>, obj: JObject<'a>, mode: ReleaseMode) -> Result<Self> {
```

And de-reference it below, on line 86:
`            env: *env,`

That way, **all the rest can stay the same** (except for removing references to `'b'` (and `'_` when required).